cmd/common.go

```diff
@@ -147,8 +77,9 @@ func filterLatestProfiles(profiles []profileutil.ProvisioningProfileInfoModel) [
 	return filteredProfiles
 }
 
-func collectIpaExportCodeSignGroups(tool Tool, archive xcarchive.IosArchive, installedCertificates []certificateutil.CertificateInfoModel, installedProfiles []profileutil.ProvisioningProfileInfoModel) ([]export.IosCodeSignGroup, error) {
-	iosCodeSignGroups := []export.IosCodeSignGroup{}
+func collectIpaExportCodeSignGroups(tool Tool, archive Archive, installedCertificates []certificateutil.CertificateInfoModel, installedProfiles []profileutil.ProvisioningProfileInfoModel) ([]export.CodeSignGroup, error) {
+	collectedCodeSignGroups := []export.CodeSignGroup{}
+	_, isMacArchive := archive.(xcarchive.MacosArchive)
```

Comment:
in case of macos export we have different possible export methods: https://github.com/bitrise-steplib/steps-xcode-archive-mac/blob/master/step.yml#L40